package/src/api-custom-element.ts

```diff
@@ -321,15 +321,58 @@ export class VueElement extends BaseClass {
       // replace slot
       if (!this._config.shadowRoot) {
         this._slots = {};
-        for (const child of Array.from(this.children)) {
-          const slotName = child.getAttribute('slot') || 'default';
+
+        const processChildNodes = (nodes: NodeListOf<ChildNode>): any[] => {
```

Comment:
Do you think we can be more explicit and not use `any` type ?
maybe something like (VNode | string)[]

Reply:
Yes, we can use `(VNode | string)[]` instead of `any` for clarity and type safety
I’ve updated it accordingly